client/src/styles/global.css

```diff
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --cyf-color: #ff1e1e;
+  --cyf-background: #ffffff;
+  --milestone-table__wrapper: #808080;
+  --user__container: #000000:
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--user__container);
+  background-color: var(--cyf-background);
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 13px;
```

Comment:
Using px values for heading is bad for accessibility also pleas take a look at the comment regarding following CYF sizes

Reply:
I used em instead of px as you said